en/docs/developer-guides/tools/migration-tools/logic-apps-migration-tool.md

```diff
@@ -0,0 +1,463 @@
+# Azure Logic Apps migration tool
```

Comment:
Add the supported versions similar to https://bi.docs.wso2.com/developer-guides/tools/migration-tools/mulesoft-migration-tool/#supported-mule-versions